test/PowerNotifications/PowerTests.dll.manifest

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+    <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
+
+    <file name="Microsoft.ProjectReunion.dll">
```

Comment:
You'll want to delete this.

If you use the Bootstrap API you can test use your WinRT objects from the test WASfwk package like the standard MSIX distribution model. See email for more details.

Also, it looks like this Fusion manifest is solely to use RegFreeWinRT to get at your WinRT API. That won't work on RS5 (w/o Reunion) so your test will fail in some of the Helix configurations (we run tests across all supported OSes, which today means RS5+). The ::TP::Bootstrap solution works on all supported platforms.

Reply:
Working on this.